music_assistant/helpers/audio.py

```diff
@@ -299,8 +302,8 @@ async def get_stream_details(
         if (
             streamdetails.stream_type in (StreamType.ICY, StreamType.HLS, StreamType.HTTP)
             and streamdetails.media_type == MediaType.RADIO
+            and isinstance(streamdetails.path, str)
```

Comment:
please remove this leftover code, its touching a core controller in a provider PR

Reply:
On this one this is actually required to make the provider work because the current code assumes all HTTP radio streams are single URLs, but this provider uses multi-part paths for sequential track streaming.

Reply:
Or should I just put it through a quick dedicated PR?